evaluation.md

```diff
@@ -0,0 +1,42 @@
+## Technical Requirements
+
+### Back-end
+- Your API includes at least 2 models, one of which may be a user
+- Your models include at least one association
+- At least one model includes all CRUD functions via RESTful routes
+
+### Front-end
+- Your app is enjoyable to use
+- Your app includes at least one view
+- Your view(s) includes at least one AJAX request that allows a user to somehow interact with the API
+- Your code, as rendered in the browser, passes a [CSS validator](http://jigsaw.w3.org/css-validator/) and an [HTML validator](https://validator.w3.org/)
+
+### Publication
+- Your code is commented only with what is necessary do document what is happening in your code; that is, there are no commented-out chunks of unused code
+- Your submission is received via a pull request to the [pbj-project3](https://github.com/ga-dc/pbj-project3) Github repository [on time](readme.md)
```

Comment:
can we create a new repo for submissions? trying to avoid issues collecting and slowing down the app